src/app/users/users.component.ts

```diff
@@ -122,7 +122,7 @@ export class UsersComponent implements OnInit, OnDestroy, AfterViewInit {
       ),
       filterDropdowns(this.filter),
       filterFieldExists([ 'doc.requestId' ], this.filterType === 'associated'),
-      filterSpecificFields([ 'doc.name' ])
+      filterSpecificFields([ 'doc.firstName', 'doc.lastName' ])

```

Comment:
@perryc85 In reviewing this, I realize that having this split in the filter means that if you search for the **first name** and **last name** together the name will not show up.  The solution is to make a change in `initializeData()` to add a property to the `userInfo` object called `fullName` with the same logic in the template.

That way the search can simply use `fullName` and will correctly return first & last name searches.